src/In.ProjectEKA.HipService/DataFlow/DataFlow.cs

```diff
@@ -6,51 +6,103 @@ namespace In.ProjectEKA.HipService.DataFlow
     using HipService.Consent;
     using Logger;
     using MessagingQueue;
+    using Microsoft.Extensions.Options;
 
     public class DataFlow : IDataFlow
     {
         private readonly IMessagingQueueManager messagingQueueManager;
         private readonly IDataFlowRepository dataFlowRepository;
         private readonly IConsentRepository consentRepository;
+        private readonly IHealthInformationRepository healthInformationRepository;
+        private readonly IOptions<DataFlowConfiguration> dataFlowConfiguration;
 
         public DataFlow(IDataFlowRepository dataFlowRepository,
             IMessagingQueueManager messagingQueueManager,
-            IConsentRepository consentRepository)
+            IConsentRepository consentRepository,
+            IHealthInformationRepository healthInformationRepository,
+            IOptions<DataFlowConfiguration> dataFlowConfiguration)
         {
             this.dataFlowRepository = dataFlowRepository;
             this.messagingQueueManager = messagingQueueManager;
             this.consentRepository = consentRepository;
+            this.healthInformationRepository = healthInformationRepository;
+            this.dataFlowConfiguration = dataFlowConfiguration;
         }
 
-        public async Task<Tuple<HealthInformationResponse, ErrorRepresentation>> HealthInformationRequestFor(
+        public async Task<Tuple<HealthInformationTransactionResponse, ErrorRepresentation>> HealthInformationRequestFor(
             HealthInformationRequest request)
         {
             var consent = await consentRepository.GetFor(request.Consent.Id);
-            if (consent == null)
-            {
-                return new Tuple<HealthInformationResponse, ErrorRepresentation>(null,
-                    new ErrorRepresentation(new Error(ErrorCode.ContextArtefactIdNotFound,
-                        ErrorMessage.ContextArtefactIdNotFound)));
-            }
+            if (consent == null) return ConsentArtefactNotFound();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=5067092)